src/components/select/select.scss

```
@@ -25,8 +25,9 @@
     
     &__value-arrow {
         position: absolute;
```

```suggestion
        // right most, vertically centered
```

add a comment for clarity. also we need DCO to re-run 😉

---

Oops I made a mistake here and accidentally removed the `position: absolute;` 😬 
Fixed in #563